.github/workflows/stale.yml

```diff
@@ -0,0 +1,70 @@
+name: Stale issues and pull-requests
+on:
+  schedule:
+    - cron: 0 * * * *
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4.0.0
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          # Idle number of days before marking issues/prs stale.
+          days-before-stale: 500
+
+          # Idle number of days before closing stale issues/prs.
+          days-before-close: 28
+
+          # Comment on the staled issues.
+          stale-issue-message: |
+            In order to maintain a backlog of relevant issues, we automatically label them as stale after 500 days of inactivity.
+
+            If this issue is still important to you, then please comment on this issue or remove the stale label.
```

Comment:
I think only core devs can remove the label.  How about "If this issue is still important to you, then please comment on this issue and the stale label will be removed." ?